src/main/java/com/citihub/configr/mongostorage/MongoMetadataStore.java

```diff
@@ -0,0 +1,42 @@
+package com.citihub.configr.mongostorage;
+
+import java.util.Optional;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import com.citihub.configr.metadata.Metadata;
+import com.citihub.configr.storage.MetadataStore;
+
+@Service("metadataStore")
```

Comment:
Eventually we can change this to a factory that will inject the implementation based on the storage model selected, but since we only support Mongo, this is easiest.

Reply:
agreed